services/records_to_agol.py

```diff
@@ -88,7 +88,7 @@ def main():
             "container_id": f"eq.{container}",
             "updated_at": f"gte.{filter_iso_date_str}",
         },
-        order_by="record_id",
+        order_by="id",
```

Comment:
I don't see a `limit` parameter here so is it paginating the results? Not really following why it's faster

Reply:
sorry, i misspoke in the PR description. we have to use a column in the `order_by` parameter that will ensure the records are always returned in an identical order _if_ postgrest client is paginating. so pagination is just enabled by default in knack services.

the gotcha is that the `record_id` column is a long string, and are ~1 million records in the `knack` record table. i had thought that the `record_id` column was indexed, but the index is only operating on the compound primary key for this table, which is `record_id`, `container_id`, and `app_id`.  we could add another index directly on the `record_id` column, but it will definitely be more performant to use an integer.

so this PR adds indexing on the `order by` column which is badly needed b/c it's applied on every query.

Reply:
as i'm revisiting this i'm realizing we should add indices for `app_id` and `container_id` as well!